nextcloud-aio-helm-chart/Chart.yaml

```
@@ -1,6 +1,6 @@
 name: nextcloud-aio-helm-chart
 description: A generated Helm Chart for Nextcloud AIO from Skippbox Kompose
-version: 11.0.0
+version: 11.1.0
```

please revert to 11.0.0. This will be done autoamtically when 11.1.0 becomes stable
```suggestion
version: 11.0.0
```

---

done